util/lua/lua.go

```diff
@@ -60,7 +61,7 @@ type VM struct {
 	UseOpenLibs bool
 }
 
-func (vm VM) runLua(obj *unstructured.Unstructured, script string) (*lua.LState, error) {
+func (vm VM) runLua(obj *unstructured.Unstructured, script string, resourceActionParameters []*applicationpkg.ResourceActionParameters) (*lua.LState, error) {
```

Comment:
Can a new method, something like `runLuaWithResourceActionParameters`, be added? And the new method would invoke the existing one and then operate on the input parameters afterwards?
This way most places in the code, that don't require the parameters, can remain unchanged and then only the new code which needs the params would invoke the new function.

Reply:
thank you for the suggestion, added a new method that invokes the existing one.